101-cognitive-services-Computer-vision-API/azuredeploy.parameters.json

```diff
@@ -0,0 +1,9 @@
+{
+  "$schema": "http://schema.management.azure.com/schemas/2015-01-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "accountName": {
+      "value": "nameofyourcomputervisionaccount"
```

Comment:
use GEN-UNIQUE

Reply:
Hi Brian,
this has not to be unique

Reply:
ok, then let's use the defaultValue in the template and remove the param value from here...

Reply:
Done

Reply:
I removed the value from the parameters file but the builds fails because the are not vaules at all in the file.